src/modules/keyboardmanager/dll/dllmain.cpp

```diff
@@ -247,33 +247,46 @@ class KeyboardManager : public PowertoyModuleIface
     // Called by the runner to pass the updated settings values as a serialized JSON.
     virtual void set_config(const wchar_t* config) override
     {
-        try
-        {
-            // Parse the input JSON string.
-            PowerToysSettings::PowerToyValues values =
-                PowerToysSettings::PowerToyValues::from_json_string(config);
-
-            // If you don't need to do any custom processing of the settings, proceed
-            // to persists the values calling:
-            values.save_to_settings_file();
-        }
-        catch (std::exception&)
-        {
-            // Improper JSON.
-        }
+        //try
+        //{
+        //    // Parse the input JSON string.
+        //    PowerToysSettings::PowerToyValues values =
+        //        PowerToysSettings::PowerToyValues::from_json_string(config);
+
+        //    // If you don't need to do any custom processing of the settings, proceed
+        //    // to persists the values calling:
+        //    values.save_to_settings_file();
+        //}
+        //catch (std::exception&)
+        //{
+        //    // Improper JSON.
+        //}

```

Comment:
Why did you comment out this? It should be used to save the `settings.json` file for KBM. Although now that I think about it, I don't think there is any settings for KBM thats changed from SettingsV2, so it probably doesn't matter anyway.

Reply:
Actually I think you're right this might be used for the settings.json file. I saw the code above where the default.json file is created and assumed that it was created there. I'll revert this change.